### DISCUSSION TITLE
Wireless settings not shown on Rpi1

### DISCUSSION THREAD

**Discussion by vale-max** [rern/rAudio#334] [Bug Reports]
Hello @rern 
I've seen you introduced a version check to show the Wireless settings in the system settings page, however this does not take in consideration that USB wifi or bt dongles can be added.
In consequence of this BT dongles can't be activated on Rpi1.

**Comment by R101G** [comment]
The problem is that Arch does not include dongle drivers in the Pi kernel. The driver for the Edimax dongle, which used to work, did not install when I last tried it, but it might have been updated now. The rtl88x2bu driver is working at the moment, and seems to have more support.

I am not sure what support there is for BT dongles, but it looks as if there are similar problems.

**Reply by vale-max** [reply]
I'm using now an inexpensive BT 5.0 dongle based on RTL I bougth on [Amazon](https://www.amazon.it/gp/product/B089Q7NJLC/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1) which is seen as 
"ID 0bda:8771 Realtek Semiconductor Corp"

The kernel from latest image has the btusb module that supports it, what was missing was the firmware, I installed package  [rtl8761b-fw](https://aur.archlinux.org/packages/rtl8761b-fw/) and the bluetooth interface works.

I haven't tried as sender but as a BT receiver it just works.
